powersimdata/design/investment/README.md

```diff
@@ -75,63 +67,38 @@ Then all costs are summed by technology (so you can ignore the non-renewables/st
 - Select (arbitrary) TechDetail (sub-technology) because we can't differentiate between these yet in our plants.
 - Map base costs to plants by "Technology".
 - Map plant location to ReEDS region (for regional cost multipliers).
-- (messy) If a technology is wind or wind_offshore or csp, regions are in rs (wind resource region), so keep rs is kept as the region to map. If technology is another tech, regions are in rb (BA region), so keep rb as the region to map.
-- Map ["Technology","r" (region)] to ReEDS regional capital cost multipliers. Keep (arbitrary) subclasses for renewables.
+- If a technology is wind or wind_offshore or csp, regions are in rs (wind resource region), so keep rs is kept as the region to map. If technology is another tech, regions are in rb (BA region), so keep rb as the region to map.
+- Map ["Technology", "r" (region)] to ReEDS regional capital cost multipliers. Keep (arbitrary) subclasses for renewables.
+- Apply inflation to scale 2018 dollars to present.
 - Final calculations.
 
-**Warning**: there's a lot of messy code here right now because I'm manually dropping a lot of subtypes of technologies so we only have one type.
-
-## Mapping functions
-
-**sjoin_nearest**: joins a geodataframe of Points and Polygons/Multipolygons. Used in `points_to_polys`.
 
-**points_to_polys**: joins a dataframe (which includes lat and lon columns) with a shapefile. Used in `bus_to_neem_reg` and `plant_to_reeds_reg`.
+### Mapping functions
 
+`sjoin_nearest`: joins a geodataframe of Points and Polygons/Multipolygons. Used in `points_to_polys`.
 
+`points_to_polys`: joins a dataframe (which includes lat and lon columns) with a shapefile. Used in `bus_to_neem_reg` and `plant_to_reeds_reg`.
 
-**Functions used for AC regional multiplier mapping**
+#### Functions used for AC regional multiplier mapping
 
-**bus_to_neem_reg**: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
+`bus_to_neem_reg`: maps bus locations to NEEM regions. Used in `write_bus_neem_map` and (if there are errors in the mapping file produced in `write_bus_neem_map`), this function is also used in `_calculate_ac_inv_costs`.
```

Comment:
Not sure I understand the above sentence. I read taht the `bus_to_neem_reg` function is used in `_calculate_ac_inv_costs` only if there are errors in the mapping file.

Reply:
The wording is a bit strange. It is used within `write_bus_neem_map` to create a mapping file, and if a cached mapping file does not map all buses successfully at runtime then it is used as a part of `_calculate_ac_inv_costs`.